WebAPI/CurrencyConverter/Interfaces/IExchangeRateRepository.cs

```diff
@@ -2,17 +2,18 @@
 using System;
 using System.Collections.Generic;
 using System.Linq.Expressions;
+using System.Threading.Tasks;
 
 namespace CurrencyConverter.Interfaces
 {
     public interface IExchangeRateRepository
     {
-        ExchangeRate Get(int id);
-        ExchangeRate GetByName(string currencyName);
-        IEnumerable<ExchangeRate> GetAll();
+        Task<ExchangeRate> Get(int id);
+        Task<ExchangeRate> GetByName(string currencyName);
+        Task<IEnumerable<ExchangeRate>> GetAll();
         IEnumerable<ExchangeRate> Find(Expression<Func<ExchangeRate, bool>> expression);
-        void Add(ExchangeRate exchangeRate);
-        void AddRange(IEnumerable<ExchangeRate> exchangeRates);
+        Task Add(ExchangeRate exchangeRate);
+        Task AddRange(IEnumerable<ExchangeRate> exchangeRates);
```

Comment:
Asynchronous methods are called with 'Async' at the end. For example - GetByNameAsync